microprofile/tests/arquillian/src/main/java/io/helidon/microprofile/arquillian/HelidonContainerConfiguration.java

```diff
@@ -16,6 +16,11 @@
 
 package io.helidon.microprofile.arquillian;
 
+import java.util.ArrayList;
```

Comment:
Please explain changes done to this module. Any change here can have unexpected impact on any TCK, so these should be handled with extreme care.

Reply:
Allows tweaking config builder from Arquillian container events, it shouldn't affect other tcks https://github.com/oracle/helidon/blob/28c81fa4464e0a601539aa5fb3054ce7d041ca4e/microprofile/tests/tck/tck-lra/src/test/java/io/helidon/microprofile/lra/tck/CoordinatorDeployer.java#L39-L83